Project.toml

```diff
@@ -1,16 +1,13 @@
 name = "Blosc"
 uuid = "a74b3585-a348-5f62-a45c-50e91977d574"
-version = "0.6.0"
+version = "0.7.0"
 
 [deps]
-BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
-CMakeWrapper = "d5fb7624-851a-54ee-a528-d3f3bac0b4a0"
+Blosc_jll = "0b7ba130-8d10-5ba8-a3d6-c5182647fed9"
 Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
```

Comment:
Don't need Libdl anymore...

Reply:
Without Libdl in deps `]test Blosc` fails
```
ERROR: LoadError: ArgumentError: Package Blosc does not have Libdl in its dependencies:
- If you have Blosc checked out for development and have
  added Libdl as a dependency but haven't updated your primary
  environment's manifest file, try `Pkg.resolve()`.
- Otherwise you may need to report an issue with Blosc
```

Reply:
Removed Libdl in an added commit